modules/isc_dhcpd/isc_dhcpd_test.go

```diff
@@ -104,7 +104,7 @@ func TestDHCPd_Check(t *testing.T) {
 	for name, test := range tests {
 		t.Run(name, func(t *testing.T) {
 			dhcpd := test.prepare()
-			dhcpd.Init()
+			require.True(t, dhcpd.Init())
 
 			if test.wantFail {
 				assert.False(t, dhcpd.Init())
```

Comment:
`assert.False(t, dhcpd.Init())` => `assert.False(t, dhcpd.Check())`, same for `else` branch